packages/webapp/src/utils/__tests__/credits-application-validation.spec.js

```diff
@@ -5,10 +5,16 @@ import applicant from '../../__mocks__/applicant'
 import creditsPurchaseConstants from '../credits-purchase-constants.js'
 import constants from '../constants.js'
 
+const populateSessionWithDevelopmentProjectInfo = (session) => {
```

Comment:
See comment from `check-and-submit.spec.js` about adding these to the mock data file